packages/actor-init-sparql-file/config/sets/rdf-source-identifiers.json

```diff
@@ -1,5 +1,6 @@
 {
   "@context": [
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-init-sparql/^1.0.0/components/context.jsonld",
```

Comment:
Why was this added?

Reply:
Because that's where the `config-sets` prefix is defined. This config did not work before apparently.